Fix/#6080 submission email

yonatan-dp opened the pull request:
Defects reported for issue 6080.

yonatan-dp committed:
Fixing defects on 6080

yonatan-dp committed:
Merge branch 'develop' into fix/#6080-submission-email

yonatan-dp committed:
Fixing defects on 6080

yonatan-dp committed:
Merge branch 'develop' into fix/#6080-submission-email

djw4erg reviewed (changes requested):
Yonatan, everything looks good except that the "main folder" version of the scripts do not seem to exist in the repository.

The scripts for changed objects should exist under the folder in the repository root with the schema name.  The scripts to implement changes should exist in the deployment folder.  Usually the whole script for non-tables and new tables, and update scripts for changed tables.

yonatan-dp committed:
Fixing defects on 6080, updates based on PR review

yonatan-dp commented:
> Yonatan, everything looks good except that the "main folder" version of the scripts do not seem to exist in the repository.
> 
> The scripts for changed objects should exist under the folder in the repository root with the schema name. The scripts to implement changes should exist in the deployment folder. Usually the whole script for non-tables and new tables, and update scripts for changed tables.

Yes, of course. I have corrected per your comment.  Also, please review camdecmps.vw_monitor_location. There was one already under the camdecmps schema root folder and I updated it to match whats in camdecmpswks. There is a slight variation in the query between what was in camdecmps and camdecmpswks.

djw4erg commented on camdecmpswks/functions/get-qatest-feedback.sql:
Sorry I missed this, but this script belongs in the CAMDECMPSWKS folder, not the CAMDECMPS folder.

yonatan-dp replied:
Hi Dwayne, the scripts are already in the CAMDECMPSWKS folder. Did you mean you want them in the CAMDECMPS folder?  I put them in the CAMDECMPSWKS folder because they are processing workspace (non official data).

djw4erg commented on camdecmpswks/functions/get-qce-feedback.sql:
Sorry I missed this, but this script belongs in the CAMDECMPSWKS folder, not the CAMDECMPS folder.

yonatan-dp replied:
Hi Dwayne, the scripts are already in the CAMDECMPSWKS folder. Did you mean you want them in the CAMDECMPS folder?  I put them in the CAMDECMPSWKS folder because they are processing workspace (non official data).

djw4erg commented on camdecmpswks/functions/get-tee-feedback.sql:
Sorry I missed this, but this script belongs in the CAMDECMPSWKS folder, not the CAMDECMPS folder.

yonatan-dp replied:
Hi Dwayne, the scripts are already in the CAMDECMPSWKS folder. Did you mean you want them in the CAMDECMPS folder?  I put them in the CAMDECMPSWKS folder because they are processing workspace (non official data).
